### PR TITLE
GDP-688 - Run automated acceptance test scripts

### PR DESCRIPTION
**Pull request by mirzak**

### CODE REVIEW

**Comment by mirzak** [comment]
This is ready to be merged @gunnarx, unless you have any review comments.

File: scripts/start-qemu-daemon [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                QEMU_SYSTEM=${QEMU_SYSTEM:-"qemu-system-x86_64"}
+                ;;
+            *86*)
+                QEMU_SYSTEM=${QEMU_SYSTEM:-"qemu-system-i386"}
+                ;;
+        esac
+
+        ;;
+    *)
+        ;;
```
Comment (gunnarx) [inline scripts/start-qemu-daemon]: Should have some error message here I think

File: scripts/start-qemu-daemon [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+set -x
+
+# if called under Yocto/OE shell, BUILDDIR will already be set correctly
+if [ -z "$BUILDDIR" ]; then
+    # try to guess BUILDDIR
+    BUILDDIR="`dirname $0`"/../gdp-src-build
+fi
+
+IMAGE_NAME=$1
+# check if it's named like an image
```
Comment (gunnarx) [inline scripts/start-qemu-daemon]: You're not checking this so I'd remove the comment.

File: scripts/start-qemu-daemon (hunk 1 above)
Comment (gunnarx) [reply scripts/start-qemu-daemon]: "Could not determine $MACHINE value or unsupported $MACHINE"?

File: scripts/start-qemu-daemon [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    -no-reboot \
+    -device nec-usb-xhci \
+    -device usb-tablet \
+    -device virtio-rng-pci  \
+    -device virtio-net-pci,netdev=net0,mac="${RANDOM_MAC}" \
+    -netdev tap,id=net0,ifname=tap0,script=no,downscript=no \
+    -append "root=/dev/hda console=ttyS0 rw mem=512M ip=192.168.7.2::192.168.7.1:255.255.255.0 oprofile.timer=1" \
+    $QEMU_ARGS
+
+exit $!
```
Comment (gunnarx) [inline scripts/start-qemu-daemon]: $! is the PID of the most recent background command.
It shouldn't be used as exit status should it?

Comment (mirzak) [reply scripts/start-qemu-daemon]: Yeah, left over of an initial "design" where PID of QEMU process was returned to the caller. But found the "pidfile" argument to qemu which is used instead.

Will clean this up.

File: scripts/start-qemu-daemon (hunk 1 above)
Comment (mirzak) [reply scripts/start-qemu-daemon]: True, will update

File: scripts/start-qemu-daemon (hunk 2 above)
Comment (mirzak) [reply scripts/start-qemu-daemon]: Will clean-up

**Comment by mirzak** [comment]
@gunnarx , updated

**Comment by gunnarx** [comment]
Great work @mirzak

**Pull request merged**